tests/functional/features/healthcare_worker_update.feature

```diff
@@ -0,0 +1,125 @@
+Feature: Healthcare Worker Access (Update)
+    Update a PDS record as a healthcare worker
+    
+    Scenario: Update patient
+        Given I am a healthcare worker
+        And I have a patient's record to update
+        And I wish to update the patient's gender
+
+        When I update the patient's PDS record
+
+        Then I get a 200 HTTP response code
+        And the response body contains the patient's new gender
+        And the response body contains the record's new version number
+
+    Scenario: Update patient using deprecated respond-async still returns 200
+        Given I am a healthcare worker
+        And I have a patient's record to update
+        And I wish to update the patient's gender
+        And I have a header Prefer value of "respond-async"
+
+        When I update the patient's PDS record
+
+        Then I get a 200 HTTP response code
+        And the response body contains the patient's new gender
+        And the response body contains the record's new version number
+
+    Scenario: Update patient with invalid wait header still updates
+        Given I am a healthcare worker
+        And I have a patient's record to update
+        And I wish to update the patient's gender
+        And I have a header X-Sync-Wait value of "invalid"
+
+        When I update the patient's PDS record
+
+        Then I get a 200 HTTP response code
+        And the response body contains the patient's new gender
+        And the response body contains the record's new version number
+
+    Scenario: Update patient with low wait header
+        Given I am a healthcare worker
+        And I have a patient's record to update
+        And I wish to update the patient's gender
+        And I have a header X-Sync-Wait value of "0.25"
+
+        When I update the patient's PDS record
+
+        Then I get a 503 HTTP response code
```

Comment:
why 503 rather than 40x?

Reply:
This was how the test was previously set up.

Looking at the [docs](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir#patch-/Patient/-id-), this appears to be the expected behaviour.